### PR TITLE
Added Dutch translation & PersonalEmail features

### PR DESCRIPTION
**Pull request by Laskewitz**
#### Category
- [ ] Bug Fix
- [x] New Feature

#### What's in this Pull Request?

Dutch language! Hope it's in time for the release!

### CODE REVIEW

**Comment by Laskewitz** [comment]
@VesaJuvonen I think I'm just a little too late with this... :-(

**Comment by erwinvanhunen** [comment]
Unfortunately you are :-( But we'll have it merged for the next release :-)

File: solution/src/extensions/discussNow/loc/nl-nl.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+define([], function() {
+    return {
+      "DiscussNowCommand": "Discuss Now",
+      
+      "DiscussNowDialogTitle": "Bespreek nu",
+      "DiscussNowDialogDescription": "Geef een onderwerp, datum en lengte van een meeting op om het geselecteerde document te bespreken",
+      
+      "ScheduleMeetingSubjectLabel": "Meeting onderwerp:",
+      "ScheduleMeetingSubjectValidationErrorMessage": "Geen een onderwerp tekst op van langer dan 10 tekens.",
```
Comment (erwinvanhunen) [inline solution/src/extensions/discussNow/loc/nl-nl.js]: small typo

File: solution/src/extensions/discussNow/loc/nl-nl.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "ScheduleMeetingDateLabel": "Meeting datum:",
+      "ScheduleMeetingHoursLabel": "Uur",
+      "ScheduleMeetingHoursValidationError": "Geef een valide uur waarde op",
```
Comment (erwinvanhunen) [inline solution/src/extensions/discussNow/loc/nl-nl.js]: Isn't "geef een geldige uur waarde op' better? 'Valide' feels like an 'anglicism'

File: solution/src/webparts/recentlyVisitedSites/loc/nl-nl.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+define([], function() {
+    return {
+      "NoRecentSitesMsg": "We hebben geen recent gebruikte sites gevonden.",
+      "Loading": "Recent gebruikte sites aan het ophalen.",
+      "Error": "Sorry, er is iets foutgegaan bij het ophalen van je recent gebruikte sites."
```
Comment (erwinvanhunen) [inline solution/src/webparts/recentlyVisitedSites/loc/nl-nl.js]: I think there needs to be a space between 'fout' and 'gegaan'

**Comment by Laskewitz** [comment]
Added some features to the PersonalEmail web part: 

- show inbox only 
You can now use a toggle in web part edit mode to make sure you only get e-mails from the Inbox instead of all e-mails (sent e-mail included).

- read/unread info
It's now possible to see the difference between read and unread emails. This feature makes it easier to see which emails you still have to take a look at and which e-mails you already have read.

**Comment by eoverfield** [comment]
@Laskewitz I suggest you create another branch and a specific PR for your personal email features. Mixing multiple updates in one PR can cause the core team delays. Also, I noticed you are using import { MSGraphClient } from '@microsoft/sp-client-preview'; for the Graph client. In the Dev branch you will notice that @waldekmastykarz updated the webparts to use SPFx 1.6, which allows you to import MSGraphClient from @microsoft/sp-http. I would recommend you consider merging your code with the latest from the dev branch for the personal email enhancements.

**Comment by Laskewitz** [comment]
@eoverfield will do from now on...

**Pull request closed without merging**